drivers/net/mlx5/mlx5_prm.h

```diff
@@ -855,19 +877,73 @@ struct mlx5_ifc_query_special_contexts_in_bits {
 	u8 reserved_at_40[0x40];
 };
 
+struct mlx5_ifc_general_obj_in_cmd_hdr_bits {
+	u8 opcode[0x10];
+	u8 reserved_at_10[0x20];
+	u8 obj_type[0x10];
+	u8 obj_id[0x20];
+	u8 reserved_at_60[0x20];
+};
+
+struct mlx5_ifc_general_obj_out_cmd_hdr_bits {
+	u8 status[0x8];
+	u8 reserved_at_8[0x18];
+	u8 syndrome[0x20];
+	u8 obj_id[0x20];
+	u8 reserved_at_60[0x20];
+};
+
+struct mlx5_ifc_virtq_bits {
+	u8 reserved_at_0[0x84];
+	u8 queue_type[0x4];
+	u8 qnum[0x18];
+	u8 desc_addr[0x40];
+	u8 used_addr[0x40];
+	u8 available_addr[0x40];
+	u8 doorbell_stride_idx[0x10];
+	u8 queue_size[0x10];
+	u8 ctrl_mkey[0x20];
+	u8 data_mkey[0x20];
+	u8 msix[0x8];
```

Comment:
since we are not covering interrupts yet, and we haven't fully discussed on it, i suggest to remove it, and maybe add later.

Reply:
ok